.coveragerc

```diff
@@ -0,0 +1,29 @@
+# .coveragerc to control coverage.py
+[run]
+branch = True
+omit =
+  .tox/*
+  *virtualenv/*
```

Comment:
What do you think about omitting `database.py`, `settings.py`, `urls.py`, and `wsgi.py`? They are a mix of boilerplate and configs and likely won't have tests written against them.

Reply:
I don't really have an opinion on it, but I don't think that should block this PR. 

We can expand the list of omissions in a future PR.